ext/intl/formatter/formatter_main.c

```diff
@@ -63,6 +64,11 @@ static int numfmt_ctor(INTERNAL_FUNCTION_PARAMETERS, zend_error_handling *error_
 		locale = intl_locale_get_default();
 	}
 
+	if (strlen(uloc_getISO3Language(locale)) == 0) {
+		zend_argument_value_error(1, "is invalid");
```

Comment:
I think it would be nice to show what the locale _is_ to say that it is invalid. So maybe something like
```suggestion
		zend_argument_value_error(1, " \"%s\" is invalid", locale);
```